inlong-manager/manager-schedule/src/main/java/org/apache/inlong/manager/schedule/dolphinscheduler/DolphinScheduleUtils.java

```diff
@@ -489,20 +493,49 @@ public static void delete(String url, String token, long code) {
             Map<String, String> header = buildHeader(token);
 
             String requestUrl = url + "/" + code;
+            for (int retryTime = 1; retryTime <= DS_DEFAULT_RETRY_TIMES; retryTime++) {
+                JsonObject response = executeHttpRequest(requestUrl, DELETE, new HashMap<>(), header);
 
-            JsonObject response = executeHttpRequest(requestUrl, DELETE, new HashMap<>(), header);
-            LOGGER.info("delete process or project success, response data: {}", response);
+                if (response.get(DS_SUCCESS).getAsBoolean()) {
+                    LOGGER.info("Delete process or project success, response data: {}", response);
+                    return;
+                }
+
+                if (response.get(DS_CODE).getAsInt() == PROCESS_DEFINITION_IN_USED_ERROR) {
```

Comment:
What to do when `response.get(DS_CODE).getAsInt() != PROCESS_DEFINITION_IN_USED_ERROR`&& response.get(DS_SUCCESS).getAsBoolean()!= true`?